test/e2e/networks/apphash-fail.toml

```diff
@@ -0,0 +1,92 @@
+# This file represents an attempt at reproducing a commonly observed failure in 
+# the generated Tendermint e2e test suite. The failure manifests as an node whose
+# AppHash as reported by /status does not agree with the AppHash as reported by the
+# block of the same height in the chain.
+
+ipv6 = true
+initial_height = 1000
+key_type = "secp256k1"
+evidence = 1
+log_level = ""
+queue_type = "priority"
+TxSize = 10240
+
+[initial_state]
+  initial01 = "a"
+  initial02 = "b"
+  initial03 = "c"
+
+[validators]
+
+[validator_update]
+  [validator_update.0]
+    validator01 = 47
+    validator02 = 93
+    validator03 = 79
+  [validator_update.1010]
+    validator04 = 63
```

Comment:
I might drop the update block unless you can confirm that this increases the likelihood of failure

Reply:
I see it in each of the apphash tests that fail so I think I'll leave it for now.